libraries-security/message.txt

```diff
@@ -0,0 +1 @@
+This is a cool message!
```

Comment:
Should this file be moved into `src/main/resources/digital-signature/` to avoid polluting the project root dir?